From 95a6611258bdcf73270451575016df0cfbf15620 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Wed, 21 Jun 2023 17:00:46 +0000
Subject: [PATCH] connman_vpn_manager.xml: Fix build with Qt 6.5

qdbusxml2cpp in Qt 6.5 no longer supports InX for out arguments
as was needed for Qt < 5.6.3 & 5.7.0 as explained in:
https://techbase.kde.org/Development/Tutorials/D-Bus/CustomTypes#Edit_the_XML

We don't care about old Qt versions, so use OutX as needed to fix:

qdbusxml2cpp: Got unknown type `a{sv}' processing '../../git/libconnman-qt/connman_vpn_manager.xml'
You should add <annotation name="org.qtproject.QtDBus.QtTypeName.Out1" value="<type>"/> to the XML description for 'properties'1

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
Upstream-Status: Pending
---
 libconnman-qt/connman_vpn_manager.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libconnman-qt/connman_vpn_manager.xml b/libconnman-qt/connman_vpn_manager.xml
index b76b8ed..18dbe5d 100644
--- a/libconnman-qt/connman_vpn_manager.xml
+++ b/libconnman-qt/connman_vpn_manager.xml
@@ -21,7 +21,7 @@
       <arg name="path" type="o" direction="in"/>
     </method>
     <signal name="ConnectionAdded">
-      <annotation name="org.qtproject.QtDBus.QtTypeName.In1" value="QVariantMap"/>
+      <annotation name="org.qtproject.QtDBus.QtTypeName.Out1" value="QVariantMap"/>
       <arg name="identifier" type="o"/>
       <arg name="properties" type="a{sv}"/>
     </signal>
